app/views/return-logs/setup/meter-provided.njk

```diff
@@ -0,0 +1,66 @@
+{% extends 'layout.njk' %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-message/macro.njk" import govukErrorMessage %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+{% block breadcrumbs %}
+  {# Back link #}
+  {{
+    govukBackLink({
+      text: 'Back',
+      href: backLink
+    })
+  }}
+{% endblock %}
+
+{% block content %}
+  {# Error summary #}
+  {% if error %}
+    {{ govukErrorSummary({
+      titleText: "There is a problem",
+      errorList: [
+        {
+          text: error.text,
+          href: "#meter-provided-error"
+        }
+      ]
+    }) }}
+  {%endif%}
+
+  {# Main heading #}
+  <div>
+    <span class="govuk-caption-l"> Return reference {{ returnReference }} </span>
+    <h1 class="govuk-heading-l">{{ pageTitle }}</h1>
+  </div>
+  <div>
+    <form method="post">
+      <input type="hidden" name="wrlsCrumb" value="{{wrlsCrumb}}"/>
+
+      {{ govukRadios({
+        name: "meterProvided",
+        errorMessage: error,
+        fieldset: {
+          legend: {
+            classes: "govuk-fieldset__legend--l"
+          }
+        },
+        items: [
+          {
+            value: "yes",
+            text: "Yes",
+            checked: "yes" == meterProvided
```

Comment:
Is there a reason why `==` has been used, shouldn't it be:
```suggestion
            checked: "yes" === meterProvided
```

Reply:
Looking at other examples in the njk files where we are comparing the checked values, we use a mix of both, there doesn't seem to be a convention on which to use

Reply:
As discussed. Since these appear to be in quite a few of the views & this isn't going live for a while, we'll have a catch with the tech lead and sort them all out at once if we need to.